### PR TITLE
Allow reloading data table

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

**Comment by bgruening** [comment]
Ah great, I just need this one! :)

**Comment by mvdbeek** [comment]
Yeah, that will make https://github.com/galaxyproject/ephemeris/blob/master/ephemeris/shed_install.py#L420 so much better :)

File: bioblend/galaxy/tool_data/__init__.py [hunk 1]
```diff
@@ -44,6 +44,16 @@ def show_data_table(self, data_table_id):
         """
         return Client._get(self, id=data_table_id)
 
+    def reload_data_table(self, data_table_id):
+        """
+        Reload a data table.
+
+        :type data_table_id: str
+        :param data_table_id: ID of the data table
```
Comment (nsoranzo) [inline bioblend/galaxy/tool_data/__init__.py]: Can you also add a description of the return value? I think it should be the same of the `show_data_table()` method.

**Comment by nsoranzo** [comment]
Thanks @mvdbeek!

**Pull request merged**